spec/services/moderator/banish_user_spec.rb

```diff
@@ -5,70 +5,39 @@
   let(:moderator) { create(:user, :trusted) }
   let(:admin) { create(:user, :super_admin) }
 
```

Comment:
This is a slow test suite. I've aggregated some of these specs together to cut down the repetitive setup time.